### PR TITLE
Remove duplicate paths.

### PR DESCRIPTION
**Pull request by kmandrika**
This come into play when a span has several subspans.

Checklist (if applicable):
- [x] Tested (manually, unit tested, etc.)
- [ ] Changelog updated
- [ ] Docs updated

### CODE REVIEW

File: js/flow/src/telemetry.ts [hunk 1]
```diff
@@ -79,13 +79,15 @@ export function writeFlowSuccess(flowName: string, latencyMs: number) {
...
+        .map((path) => [path.path, path])
     );
 
     logger.logStructured(`Paths[/${flowName}]`, {
       flowName: flowName,
-      paths: relevantPaths.map((p) => p.path),
+      paths: [...relevantPaths.keys()],
     });
 
     relevantPaths.forEach((p) => {
```
Comment (andrewbrook) [inline js/flow/src/telemetry.ts:93]: I don't think this takes into account the new data structure

**Review by andrewbrook — CHANGES_REQUESTED** [review]
Add unit tests?

File: js/flow/src/telemetry.ts (hunk 1 above)
Comment (kmandrika) [reply js/flow/src/telemetry.ts:93]: Could you say more here, not sure what you mean

Comment (andrewbrook) [reply js/flow/src/telemetry.ts:93]: Ah, I didn't expect that first arg of forEach was the entry value. So I guess this is fine.

**Comment by kmandrika** [comment]
> Add unit tests?

On it!

**Comment by kmandrika** [comment]
Thinking more about this & playing around with test cases, I don't think removing duplicate paths makes sense any more.

In cases (e.g. tool calls) where a parent span can have multiple subspans (some of which have the same path), the traversal of those paths should be counted because it happened.

Going to close this for now.
